packages/commonwealth/client/scripts/controllers/server/polls.ts

```diff
@@ -101,9 +100,9 @@ class PollsController {
       },
       success: (response) => {
         const modeledPoll = modelFromServer(response.result);
-        const thread = app.threads.getById(threadId);
-        const updatedThread = new Thread({ ...thread, hasPoll: true });
-        app.threads.updateThreadInStore(updatedThread);
+        // TODO: updateThreadInAllCaches should not be used anywhere outside of the /api/state folder
+        // This is an exception until polls get migrated to react query
+        updateThreadInAllCaches(app.activeChainId(), threadId, { hasPoll: true })
```

Comment:
`updateThreadInAllCaches` is used here but it should not be used outside of the `state/api/threads` folder (i.e outside of reactive code) - I used here as an exception until we move the polls to react query